static/admin-widget-style.css

```diff
@@ -0,0 +1,85 @@
+/* WIDGET STYLES */
+.smaily-layout-container {
+    background-color: #f6f7f7;
+    border: 1px solid #c3c4c7;
+    border-radius: 5px;
+    padding: 0px 15px 0px 15px;
+    margin-bottom: 10px;
+}
+
+/* Clear floats after the columns */
+.row {
+    display: flex;
+}
+
+/* Layout styles*/
+.smaily-layouts {
+    width: 90%;
+}
+
+/* Button styles */
+.column {
+    flex: 50%;  
+ }
+ 
+.button-color {
+    width: 95%;
+    border: 1px solid #c3c4c7;
+    border-radius: 5px;
+    padding: 5px 0px 5px 0px;
+    margin-top: 3px;
+    font-size: 14px;
+    border-color: #8c8f94;
+}
+
+.button-text-color {
+    width: 95%;
+    border: 1px solid #c3c4c7;
+    border-radius: 5px;
+    padding: 5px 0px 5px 0px;
+    margin-top: 3px;
+    font-size: 14px;
+    border-color: #8c8f94;
+}
```

Comment:
These classes are exactly the same. Try to combine these two different class names into one an apply it to both color fields.